### ISSUE TITLE
 Logistic regression training on IMDB movie reviews with 10,000 bag-of-words (BoW) feature vectors.

### ISSUE THREAD

**Issue by huangdian-scu** [bentrevett/pytorch-sentiment-analysis#91]
I would appreciate it if you offering pytorch version of Logistic regression training on IMDB movie reviews with 10,000 bag-of-words (BoW) feature vectors, which is the experiment of adam and I have problem reproducing it.   (We pre-process the IMDB movie reviews into bag-of-words (BoW) feature vectors including the first 10,000 most frequent words.  50% dropout noise can be applied to the BoW features during training to prevent over-fitting). I will express my hearty appreciation to you in my thesis. thank you!
